### PR TITLE
Add Pre-Commit Hook

### PR DESCRIPTION
**Pull request by Bendomey**

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1 @@
+yarn format && git add .
```
Comment (Bendomey) [inline .husky/pre-commit:1]: Main Addition

File: package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "npm-run-all": "4.1.5"
+  },
+  "scripts": {
+    "prepare": "husky",
+    "format": "run-s format:*",
+    "format:frontend-client": "cd apps/client && yarn format",
+    "format:frontend-waitlist": "cd apps/wait-list && yarn format",
+    "format:services-main": "cd services/main && make lint"
+  }
+}
```
Comment (Bendomey) [inline package.json:13]: Main Addition

File: services/main/Makefile [hunk 3]
```diff
@@ -18,4 +18,7 @@ build-release:
 	dotnet publish -c Release -o out
 
 build-docker:
-	docker build -t mfoni-api -f Dockerfile .
\ No newline at end of file
+	docker build -t mfoni-api -f Dockerfile .
+
+lint:
+	dotnet format main.sln
```
Comment (Bendomey) [inline services/main/Makefile:24]: Main Addition

File: package.json (hunk 2 above)
Comment (arkhurst) [reply package.json:13]: all right

**Review by arkhurst — APPROVED** [review]

**Pull request merged**